common/exception/ErrorMessage.java

```diff
@@ -41,8 +41,8 @@ public static class Server extends ErrorMessage {
                 new Server(2, "The expected data directory '%s' is not writable.");
         public static final Server EXITED_WITH_ERROR =
                 new Server(3, "Exited with error.");
-        public static final Server UNCAUGHT_EXCEPTION =
-                new Server(4, "Uncaught exception thrown at thread '%s':\n'%s'");
+        public static final Server UNCAUGHT_ERROR =
```

Comment:
we print this for both errors and exceptions, which are both errors from the user's point of view